e2e/run

```diff
@@ -156,13 +163,17 @@ customize_cluster_yaml() {
 worker:
   nodePools:
   - name: asg1
+    clusterAutoscalerSupport:
+      enabled: true
```

Comment:
@redbaron Yes, we can. kube-aws as of today already supports `controller.clusterAutoscalerSupport.enabled` to provide appropriate iam permissions to controller nodes. It would be a matter of just adding appropriate node labels and node selectors to stick cluster-autoscaler to whatever nodes(worker or controller) clusterAutoscalerSupport is enabled.